README.md

```diff
@@ -379,6 +379,28 @@ payload must conform to the following JSON schema:
 }
 ```
 
+For code from [example](#example) use:
+```sh
+curl -H "Content-Type:application/json" -XPOST -d '{"query": "{ add(x: 2, y: 2) }"}' http://localhost:3000/graphql
+```
+
+#### POST /graphql with Content-type: application/graphql
+
+Executes the GraphQL query or mutation described in the body. `operationName` and `variables` can not be passed using this method. The
+payload must conform to the following JSON schema:
+
+```js
+{
+  type: 'string',
+  description: 'the GraphQL query'
+}
+```
+
+For code from [example](#example) use:
+```sh
+curl -H "Content-Type:application/graphql" -XPOST -d "{ add(x: 2, y: 2) }" http://localhost:3000/graphql
```

Comment:
This body is not conforming to the description above. It does not respect the JSON schema

Reply:
I think JSON shema was OK. It accepts only a string that is a GraphQL query or mutation.
"{ add(x: 2, y: 1)}" equal to "query { add(x: 2, y: 1)}". Curly braces are part of the query. 

Internally it is transformed to { "query": "query { add(x: 2, y: 1)}" } before execution.

Reply:
A graphql query is not a JSON, so the content of the body cannot follow the JSON schema from a user perspective.

Reply:
Then I will remove the reference to JSON schema validation.

Reply:
I have removed JSON schema validation from POST application/graphql section.